packages/manager/modules/manager-pci-common/package.json

```diff
@@ -1,70 +1,90 @@
 {
   "name": "@ovh-ux/manager-pci-common",
   "version": "0.9.1",
-  "private": true,
-  "sideEffects": true,
+  "private": false,
+  "description": "Public Cloud Common components",
+  "homepage": "https://github.com/ovh/manager/blob/master/packages/manager/modules/manager-pci-common/README.md",
+  "bugs": {
+    "url": "https://github.com/ovh/manager/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/manager-pci-common"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
   "type": "module",
-  "main": "./src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "build": "tsc --project tsconfig.build.json",
-    "dev": "tsc",
+    "build": "vite build",
+    "clean": "rimraf node_modules && rimraf dist && yarn cache clean",
+    "dev": "vite build",
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
+    "prepare": "vite build",
+    "prettier": "prettier --write \"src/**/*.{ts,tsx,js,mdx}\"",
     "test": "vitest run",
     "test:coverage": "vitest run --coverage",
-    "test:coverage:watch": "vitest --coverage",
-    "test:watch": "vitest"
+    "test:watch": "vitest --watch",
+    "type:check": "tsc --noEmit"
   },
   "dependencies": {
+    "@ovh-ux/manager-tailwind-config": "^0.2.1",
```

Comment:
devDEp ?

Reply:
No. We need to bundle this package as it's a private one, so it has to embedded in the minified code.